content/en/monitors/service_level_objectives/_index.md

```diff
@@ -166,6 +166,28 @@ After creating your SLO, you can use the SLO Summary dashboard widget to visuali
 
 Status corrections allow you to identify specific time periods, such as planned maintenance, that an SLO should ignore from its calculation of status and error budget. Data during excluded time periods is not taken into account. A public beta for corrections is available through the [SLO status corrections API][12] and [Terraform][13].
 
+## SLO status corrections
+
+Status corrections allow you to identify specific time periods, such as planned maintenance, that an SLO should ignore from its calculation of status and error budget. 
```

Comment:
```suggestion
Status corrections allow you to specify time periods such as planned maintenance so that SLOs exclude the time from status and error budget calculations.
```